packages/host/src/Host.tsx

```diff
@@ -182,106 +183,113 @@ const Host: FC = () => {
-                              <Route
-                                path={RouteBuilder.create(AppRoute.Dashboard)
-                                  .addWildCard()
-                                  .build()}
-                                element={<DashboardService />}
-                              />
-                              <Route
-                                path={RouteBuilder.create(AppRoute.Catalogue)
-                                  .addWildCard()
-                                  .build()}
-                                element={<CatalogueContainer />}
-                              />
-                              <Route
-                                path={RouteBuilder.create(AppRoute.Distribution)
-                                  .addWildCard()
-                                  .build()}
-                                element={<DistributionContainer />}
-                              />
-                              <Route
-                                path={RouteBuilder.create(AppRoute.Suppliers)
-                                  .addWildCard()
-                                  .build()}
-                                element={
-                                  <Heading locale={locale}>suppliers</Heading>
-                                }
-                              />
-                              <Route
-                                path={RouteBuilder.create(AppRoute.Stock)
-                                  .addWildCard()
-                                  .build()}
-                                element={
-                                  <Heading locale={locale}>stock</Heading>
-                                }
-                              />
-                              <Route
-                                path={RouteBuilder.create(AppRoute.Tools)
-                                  .addWildCard()
-                                  .build()}
-                                element={
-                                  <Heading locale={locale}>tools</Heading>
-                                }
-                              />
-                              <Route
-                                path={RouteBuilder.create(AppRoute.Reports)
-                                  .addWildCard()
-                                  .build()}
-                                element={
-                                  <Heading locale={locale}>reports</Heading>
-                                }
-                              />
-                              <Route
-                                path={RouteBuilder.create(AppRoute.Messages)
-                                  .addWildCard()
-                                  .build()}
-                                element={
-                                  <Heading locale={locale}>messages</Heading>
-                                }
-                              />
+          <OmSupplyApiProvider url={Environment.API_URL}>
```

Comment:
this is the only change here 🙄 

I can see passing any other auth details here also if needed